recipes/meshplot/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "meshplot" %}
+{% set version = "v0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/skoch9/meshplot/archive/{{ version }}.tar.gz
+  sha256: 4637d49fecfd5a61bfdbfc22c0b23f7a698ae8336e02296b1f442232559c0566
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - pythreejs
+    - numpy
+    - matplotlib
+    - ipywidgets
```

Comment:
Are pythreejs, numpy, ipywidgets, and matplotlib needed for build?

Reply:
Probably not, but this is host.

Can you point/explain what is the difference between build host and run?

Reply:
Here are the docs: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements-section